test/test_performance.py.in

```diff
@@ -65,21 +71,16 @@ def _raw_to_png(csv_data, png_path):
                'latency_mean (ms)',
                'latency_variance (ms) * 100',
                'maxrss (Mb)']].plot(x='T_experiment', secondary_y=['maxrss (Mb)'])
-
-    plt.title('@TEST_NAME@ Performance Test')
-
+    plt.title('Performance tests latency\n@TEST_NAME@')
     plt.savefig(png_path)
 
     dataframe[['T_experiment', 'cpu_usage (%)']].plot(x='T_experiment')
-
-    plt.title('@TEST_NAME@ CPU usage (%) Test')
-
+    plt.title('Performance tests CPU usage (%)\n@TEST_NAME@')
+    plt.ylim(0, 100)
```

Comment:
Why are you locking the Y-axis here?

Reply:
CPU usage is a percentage so should be bounded between 0 and 100, right?

Reply:
My thought is that if all of the data is between, say, 95% and 97%, the graph is going to look like a straight line. The graph isn't very useful when that happens.

Reply:
removed `plt.ylim(0, 100)`